sugondat/chain/pallets/length-fee-adjustment/src/lib.rs

```diff
@@ -139,6 +140,30 @@ pub mod pallet {
     pub type TargetBlockSize<T: Config> =
         StorageValue<_, Perquintill, ValueQuery, TargetBlockSizeDefault>;
 
+    /// Genesis config for setting up `NextLengthMultiplier` and `TargetBlockSize` storage values.
+    /// Set None if the default values `NextLengthMultiplierDefault` & `TargetBlockSizeDefault` are to be used.
```

Comment:
Having the `_Default` members was a workaround for _not_ having a genesis config and migration. They could be removed in the future. The fields of `GenesisConfig` should not be `Option`s.

Reply:
Done